resources/js/components/attendance/AttendanceKiosk.vue

```diff
@@ -295,13 +310,15 @@
                             } else if (response.data.users.length == 1 && response.data.users[0].roles.filter(role => role.name.toString() === "admin").length === 1) {
                                 // Roles retrieved and the user is an admin
                                 console.log('User is an admin!');
```

Comment:
can we remove this line?

Reply:
Is this referring to the admin log? Can't tell, GitHub is being silly.